src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderIdentifier.java

```diff
@@ -63,6 +63,9 @@ public static class OffenderIdentifierPK implements Serializable {
     @Column(name = "CASELOAD_TYPE")
     private String caseloadType;
 
+    @Column(name = "ISSUED_AUTHORITY_TEXT")
+    private String issuedAuthorityText;
```

Comment:
is this always `INST`? Just wondering why we now need it

Reply:
This seems to contain free text entered by users - things like 'Prisoner has 2 PNCs' and 'This is the wrong PNC' wehre there are multiple different PNCs. So it seemed good to expose this text to clients.